lib/report.js

```diff
@@ -237,27 +260,6 @@ class Report {
     return maybeV8ProcessCov && Array.isArray(maybeV8ProcessCov.result)
   }
 
-  /**
-   * Returns the list of V8 process coverages generated by Node.
-   *
-   * @return {ProcessCov[]} Process coverages generated by Node.
-   * @private
-   */
-  _loadReports () {
```

Comment:
Any change this `_loadReports` could be kept here, and its API as is? This is something Vitest uses internally. https://github.com/vitest-dev/vitest/blob/708b10fe227c04b4219a1e5d4b539def009729b4/packages/coverage-c8/src/provider.ts#L94-L95

I've been meaning to open a PR here and propose this method as part of public API.

Reply:
Hmm ok I can keep loadReports but this PR fundamentally changes this in that it does not load all reports at once because of the OOM issues.

Reply:
I think maybe I can provide this async merging as a CLI arg and keep existing functionality but that means for libraries that overrode "private" methods you'd have another one to override

Reply:
hmm. I could look into that. I realized I requested a review from you but I really would like @bcoe to take a look at this.  Is this project still alive? there's quite a lot of PRs in the queue